src/pg/decoder.cr

```diff
@@ -501,7 +501,8 @@ module PG
       end
     end
 
-    @@decoders = Hash(Int32, PG::Decoders::Decoder).new(ByteaDecoder.new)
+    # :nodoc:
+    class_getter decoders = Hash(Int32, PG::Decoders::Decoder).new(ByteaDecoder.new)
```

Comment:
I did this because I was going to use decoders as the source for oids, but I didn't end up using it. I can probably undo this change.